back-office/shop-parameters/order-settings/order-settings.md

```diff
@@ -0,0 +1,111 @@
+# Order settings specs
+
+## General
+
+### Enable final summary
+
+This toggle is on No by default. 
+
+When enabled, a summary is displayed in the Front-Office during the checkout, at the payment step:
+
+![order final summary](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Order_final_summary.png)
+
+
+This way, the customer can verify before the payment all the informations: the delivery address, the invoice address, the selected shipping method & its cost, the products in the cart, the unit price of each product, the quantity of each product, the total for each product line, the subtotal, the shipping cost and the order total.
+
+### Enable guest checkout
+
+This toggle is on Yes by default.
+
+**When enabled**, guest visitors are allowed to place an order without creating an account
+
+During the checkout in front-office, at the first step (personal information), a tab "order as a guest" is displayed with the following form:
+
+![order as a guest](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Order%20as%20a%20guest.png)
+
+As the creation of an account is optional, the password field is also optional.
+
+**When disabled**, guest visitors are not allowed to place an order without creating an account.
+
+During the checkout in front-office, at the first step (personal information), a tab "Create an account" is displayed with the following form:
+
+![order create an account](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Order_create_account.png)
+
+As the creation of an account is not optional, the password field is required.
+
+### Disable reordering option
+
+This toggle is on No by default.
+
+When enabled, a "reorder" link is displayed in the order history page of the customer account and in the order detail:
+
+![reorder list](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Reorder.png)
+
+![reorder detail](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Reorder_detail.png)
+
+### Minimum purchase total required in order to validate the order
+
+By default this field is set to 0.00
+
+The merchant can enter the minimum value (taxes excluded) required in the cart for the customer to be able to make an order
+
+If the value is > 0.00 and not reached in the shopping cart front-office, then a message is displayed in the shopping cart: "A minimum shopping cart total of XX (tax excl.) is required to validate your order. Current cart total is XX (tax excl.)." and the "Proceed to checkout" button is disabled.
+
+![minimum order value](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/minimum_order_value.png)
+
+**Please note this value is not taken into account for an order created in the back-office:** the merchant is allowed to create an order with an amount < to the value defined. 
+
+### Recalculate shipping costs after editing the order
```

Comment:
Does it have an impact on the invoice?

Reply:
If there was already an invoice for the order, it doesn't impact the invoice but not sure if this is the wanted behavior @MatShir ?

Reply:
If the cart is recalculated, it is because the merchant wanted to change the order. The invoice must follow this change